### ISSUE TITLE
Support for SketchUp Make (free)?

### ISSUE THREAD

**Issue by SimonEast** [Eneroth3/eneroth-solid-tools#12]
Hi there. I'd love to use this in a personal project I'm doing with SketchUp Make (free version), but your plugin page seems to indicate it only works with the "Pro" version. Is there a technical reason for this? Or just a decision to only support the Pro version?

**Comment by LaureFR38** [comment]
I am also interested !
But I think the answer to the question can be found here :

    # Extension Warehouse doesn't allow hosting this extension unless it's
    # limited to SketchUp Pro, since it replicates a Pro feature.

**Comment by Eneroth3** [comment]
Yup, this limitation is because of the Extension Warehouse user agreement. You can download the plugin here from GitHub and remove the check for Pro.

**Comment by LaureFR38** [comment]
@Eneroth3 Yes, good idea but can you tell me in a few words how to easily build an .rbz in Windows?

**Comment by LaureFR38** [comment]
Good idea. Thank you so much.
For Windows users, go here: C: \ Users \ xxxxxx \ AppData \ Roaming \ SketchUp \ SketchUp 2017 \ SketchUp \ Plugins \ ene_solids and modify the menu.rb file ;-)

**Comment by Longjohn1** [comment]
This is superb solid tools for 2017
